Makefile

```diff
@@ -441,10 +441,12 @@ gettext:
 	# the string marked for translation beginning with "translators" will be
 	# included in the pot file.
 	$(PYTHON) ./setup.py gettext
+	${PYTHON} ./syspurpose/setup.py gettext
```

Comment:
This needs to be run from the syspurpose directory otherwise the new strings for syspurpose are not gathered.